AppBuilder/platform/views/ABViewMenu.js

```diff
@@ -23,738 +23,738 @@ module.exports = class ABViewMenu extends ABViewMenuCore {
     * @param {string} mode what mode are we in ['block', 'preview']
     * @return {Component}
     */
-   editorComponent(App, mode) {
-      var idBase = "ABViewMenuEditorComponent";
-      var ids = {
-         component: App.unique(`${idBase}_component`),
-         pages: App.unique(`${idBase}_pages`),
-         tree: App.unique(`${idBase}_tree`),
-      };
-
-      var component = this.component(App);
-
-      var menu = component.ui;
-      if (component.ui.elements) {
-         var menuIndex = 0;
-         component.ui.elements.forEach((elem) => {
-            if (elem.view == "menu") {
-               menu = component.ui.elements[menuIndex];
-            }
-            menuIndex++;
-         });
-      }
-      menu.id = ids.component;
-
-      var _ui = {
-         type: "space",
-         rows: [menu, {}],
-      };
-
-      var _init = (options) => {
-         var Menu = $$(ids.component);
-
-         this.ClearPagesInView(Menu);
-         if (this.settings.order && this.settings.order.length) {
-            this.AddPagesToView(Menu, this.settings.order);
-            // } else if (this.settings.pages && this.settings.pages.length) {
-            //    this.AddPagesToView(Menu, this.settings.pages);
-         }
-      };
-
-      var _logic = {};
-
-      return {
-         ui: _ui,
-         init: _init,
-         logic: _logic,
-      };
-   }
+   // editorComponent(App, mode) {
+   //    var idBase = "ABViewMenuEditorComponent";
```

Comment:
So we need to keep all this code? My preference is to remove it rather than leave it commented out.